js/console.ts

```diff
@@ -755,6 +758,21 @@ export class Console {
     clearScreenDown(stdout);
   };
 
+  trace = (...args: unknown[]): void => {
+    const message = stringifyArgs(args, {
+      indentLevel: this.indentLevel,
+      collapsedAt: this.collapsedAt,
+      noTrailingNewline: true
```

Comment:
Why is this necessary?

Reply:
It is just that without this settings there will be a newline appended to message by `stringifyArgs` and thus leaving a blank line between message and trace on printing `err.stack`.

(There is definitely no specification saying this is not allow, just I feel it would be kind of ugly...)

Reply:
No, I appreciate it being formatted as common. 
I suggest I'm suggesting that instead of adding noTrailingNewline, you make it so stringifyArgs doesn't add a trailing \n and lift that into the callers.

Reply:
I reviewed the structure of `console.ts` and realized that there are some complexity introduced by `console.groupCollapsed()`. I believe the original intention of introducing it was to allow synchronous write, which is available now since we added `File::writeSync`.

Technically `console.groupCollapsed()` behavior we have is non-standard, as the definition of being `collapsed` refers to something like a dropdown. Node simply aliases `groupCollapsed` to `group`